my-app/src/pages/homePage/ArticlePage.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useContext} from 'react'
+import {useParams, Link} from 'react-router-dom'
+import styled from 'styled-components'
+import {ArticleContext} from "./BlogHomeContext";
+
+
+export const ArticlePage = () => {
+    const logic = useContext(ArticleContext)
+
+    const {id} = useParams()
+    const blog = logic.postsArray.find(post => post._id === id)
+    const date = () => {

```

Comment:
shouldn't be a function

`const date = blog && new Date(blog.createdAt).toLocaleDateString('en-US')`

Reply:
fixed